blog/views/admin_post_views.py

```
@@ -0,0 +1,35 @@
+from django.contrib.admin.views.decorators import staff_member_required
+from django.shortcuts import render, redirect
+from django.http import HttpResponse
+
+from blog.forms import PostCreateForm
+from blog.services.post_create_service import create_post
+
+
+# 관리자 게시물 목록 조회
+@staff_member_required
+def admin_post_list_view(request):
```

Ruff (0.15.5)

[warning] Unused function argument: `request`

(ARG001)